spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindRefComparison.java

```diff
@@ -909,6 +913,31 @@ private void reportBest(ClassContext classContext, Method method, LinkedList<War
         }
     }
 
+    /**
+     * Identifies if it's a comparison of two instances of the class inside
+     * its equals method or if it's a comparison of the class with Object.
+     *
+     * This should be OK as this is usually an optimization inside the equals
+     * method.
+     * @param jClass: the class where this check is running on
+     * @param method: the method where the comparison is happening
+     * @param lhsType: the type of the left hand side of the comparison
+     * @param rhsType: the type of the right hand side of the comparison
+     * @return
```

Comment:
Can you please add a description to this javadoc tag?

Reply:
added as part of [df5db48](https://github.com/spotbugs/spotbugs/pull/2988/commits/df5db48bf8190dfff44297d9b823c63b2606b199)